src/zenml/utils/pipeline_docker_image_builder.py

```diff
@@ -426,7 +426,8 @@ def _gather_requirements_files(
         integration_requirements = set(
             itertools.chain.from_iterable(
                 integration_registry.select_integration_requirements(
-                    integration
+                    integration_name=integration,
+                    target_os=docker_settings.target_os,
```

Comment:
As mentioned on the other comment, I would hardcode this instead of exposing it on the docker settings.